### ISSUE TITLE
🦀 Rust Rewrite

### ISSUE THREAD

**Issue by JamieMason** [JamieMason/syncpack#222]
## Description

This issue is an FYI that most current issues are on hold while a rewrite at [v14-alpha](https://github.com/JamieMason/syncpack/tree/v14-alpha) is being done. Some of the open issues have been fixed in that branch and I'm hoping to address most of the others as part of that work as well.

I'll update each issue when this work is complete.

Note that the `prompt` and `update` commands will not be in the first release, so issues related to those commands will take a while before they are resolved.

### Support this project

[![Github-sponsors](https://img.shields.io/badge/sponsor-30363D?style=for-the-badge&logo=GitHub-Sponsors&logoColor=#EA4AAA)](https://github.com/sponsors/JamieMason)

[![support on ko-fi](https://ko-fi.com/img/githubbutton_sm.svg)](https://ko-fi.com/C0C4PY4P)

**Comment by JamieMason** [comment]
The first release of this is available at [14.0.0-alpha.1](https://github.com/JamieMason/syncpack/releases/tag/14.0.0-alpha.1)

**Comment by hirbod** [comment]
Just sliding in to motivate you—I hope you continue this project. It's beautiful. I have no idea how I managed to mess around with *npx lerna-update-wizard* for two years, screaming every time I used it, but yours is pure gold—so delightful and amazing to use. I'm still on v13, but I'd love to see you keep working on it!

**Comment by JamieMason** [comment]
Thanks Hirbod that's really great of you, little things like this are rare and they do keep us going, so thank you again I really appreciate that. I'm on a _bit_ of a short break from syncpack as the Rust rewrite was a really long period of working on nothing but this day after day. As a break from it I'm [building a PWA while also looking for a new job](https://www.linkedin.com/posts/jamiemasonleeds_opentowork-frontend-activity-7287445122466611200-K0Ib?utm_source=share&utm_medium=member_desktop), but I plan to be back on syncpack again soon.

**Comment by JamieMason** [comment]
Thanks @hirbod appreciate it, I'm completely front end though but thanks a lot for asking.

**Comment by JamieMason** [comment]
Closing as this is done and in alpha, other issues are now unblocked.
